src/display/canvas.js

```diff
@@ -856,7 +856,10 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
 
       if (this.transparentCanvas) {
         this.ctx = this.compositeCtx;
+        this.ctx.save();
```

Comment:
that `save` is called on `this.transparentCanvas` but in here we want `this.compositeCtx` have identity transform